blueflood-core/src/main/java/com/rackspacecloud/blueflood/service/CoreConfig.java

```diff
@@ -141,7 +141,8 @@
     STRING_METRICS_DROPPED("false"),
     TENANTIDS_TO_KEEP(""),
 
-    USE_ES_FOR_UNITS("false");
+    USE_ES_FOR_UNITS("false"),
+    ES_UNIT_THREADS("50"); // Should at least be equal to the number of the netty worker threads
 
```

Comment:
@GeorgeJahad Netty threads' default config currently comes from HttpConfig, and this functionality of getting units from ES lies in RollupHandler, which is base class for every protocol specific rollup handler like `HttpRollupQueryHandler`. Using the same netty config, will mean adding a dependency on HttpConfig, leading to circular dependencies between http and core modules. Also, there is no need to tightly couple it to netty thread pool, as it can be used for other protocol query handlers as well.